tests/cmdline/commands/test_computer.py

```diff
@@ -241,6 +246,48 @@ def test_noninteractive_optional_default_mpiprocs_3(self):  # pylint: disable=in
         self.assertIsInstance(result.exception, SystemExit)
         self.assertIn('mpiprocs_per_machine, must be positive', result.output)
 
+    def test_noninteractive_optional_default_mem(self):  # pylint: disable=invalid-name
+        """
+        Check that if is ok not to specify mpiprocs-per-machine
+        """
+        options_dict = generate_setup_options_dict({'label': 'computer_default_mem'})
+        options_dict.pop('default-memory-per-machine')
+        options = generate_setup_options(options_dict)
+        result = self.cli_runner.invoke(computer_setup, options)
+
+        self.assertIsNone(result.exception, result.output[-1000:])
+
+        new_computer = orm.Computer.objects.get(label=options_dict['label'])
+        self.assertIsInstance(new_computer, orm.Computer)
+        self.assertIsNone(new_computer.get_default_memory_per_machine())
+
+    def test_noninteractive_optional_default_mem_2(self):  # pylint: disable=invalid-name
```

Comment:
These tests were just converted to `pytest` style. Please adopt the same style after rebasing.

Reply:
done